python/pyspark/ml/regression.py

```diff
@@ -708,7 +710,7 @@ def _create_model(self, java_model):
 
 
 @inherit_doc
-class DecisionTreeModel(JavaModel):
+class DecisionTreeModel(JavaModel, JavaPredictionModel):
```

Comment:
I was going to leave the ones which aren't base classes for classification models for whoever picks up update the regression models (`DecisionTreeModel` is the base for `DecisionTreeClassificationModel` so I updated it).